wagtail_localize/views/report.py

```diff
@@ -111,8 +120,19 @@ class TranslationsReportFilterSet(WagtailFilterSet):
         field_name="source__specific_content_type", label=gettext_lazy("Content type")
     )
     source_title = SourceTitleFilter(label=gettext_lazy("Source title"))
-    source_locale = LocaleFilter(field_name="source__locale")
-    target_locale = LocaleFilter()
+    source_locale = LocaleFilter(
+        field_name="source__locale",
+        choices=_get_locale_choices,
+        empty_label=None,
+        null_label="All",
```

Comment:
needs to be translated

Reply:
My bad, I forgot that :facepalm: 

Do I need to translate anything from `get_content_language` as well? My assumption is that this must be handled from Wagtail/settings side

Reply:
considering get_content_language pull stuff from settings, that is up to the implementer, not us